addons/analytic/i18n/analytic.pot

```diff
@@ -209,6 +209,13 @@ msgstr ""
 msgid "Analytical Plans"
 msgstr ""
 
+#. module: analytic
+#. odoo-python
+#: code:addons/analytic/models/analytic_plan.py:0
+#, python-format
+msgid "The parent and child analytic plans must belong to the same company. Please ensure that their company information is updated accordingly."
```

Comment:
I don't see this message anywhere in the code?

Reply:
Hello @william-andre 
My bad, after changing the code I forgot to remove changes in the .pot file.